public/app/core/components/NavBar/NavBar.tsx

```diff
@@ -50,12 +50,28 @@ export const NavBarUnconnected = React.memo(({ navBarTree }: Props) => {
     setShowSwitcherModal(!showSwitcherModal);
   };
   const navTree = cloneDeep(navBarTree);
-  const topItems = navTree.filter((item) => item.section === NavSection.Core);
-  const bottomItems = enrichConfigItems(
+  let topItems = navTree.filter((item) => item.section === NavSection.Core);
+  topItems = topItems.filter((item) => {
```

Comment:
Just making sure it's captured somewhere, but we should disable menu items from the `.ini` where possible. If there isn't an `ini` toggle for disabling a menu item, we should consider adding such a toggle in a `pub/ni/*` branch and trying to upstream that change. The fact that a lot of this stuff can be toggled from the `.ini` already should embolden us to make additional things toggleable in that way.